### PR TITLE
Added multidomain switcher

### LINKED ISSUE #325: Als beheerder kan ik meerdere domeinen beheren en uitbreiden. Het is mogelijk om nieuwe domeinen toe te voegen ...
...en er kan makkelijk geswitched worden tussen de verschillende domeinen (Multi domein). De inschrijver omschrijft hoe aan deze eis voldoen kan worden, en op welke manier het multitenancy vraagstuk architectuurtechnisch en budgettair opgelost gaat worden.

Eis: Hosting 7.
Zie ook NvI: 57, 98, 106, 110, 114, 115, 184

**Tasks**

Some organizations manage multiple sites (instances of Open Forms). The admins should not need to login to each of the managed sites everytime.

- [x] Add site selectbox (showing configured sites, this can be an environment variable containing a list of URL+domainnames) in admin; on change, redirect user to selected site.
- [x] If user is logged in on current site, automatically login user on the other site (selected in the pulldown)

I see 2 directions:

1. We setup Keycloak and use OpenID-connect for admin logins.

- [x] @sjoerdie Can you install Keycloak on our test environment
- [x] @stevenbal Can you add OpenID-connect for the admin
- [x] @stevenbal Can you connect 2 Open Forms (Duiven and Zevenaar) to this Keycloak instance?

2. We use an OTP-link to switch from one site to another.

If option 1 is not working as intended, we can use this approach.

### PR DESCRIPTION
**Pull request by joeribekker**
Fixes #325 

This softly depends on https://github.com/maykinmedia/mozilla-django-oidc-db/pull/14

* It's a pretty simplistic way of adding multiple sites with the flexibility to set different URLs (depending on login-mechanism).
* It's not smart in the way that you need to set the current site yourself rather then doing semi-smart things like comparing domains, which can lead to issues with cnames and so on. It does however add the `?next` param to redirect to the same page as on the current instance.
* Cache was not added to allow the `?next` param to work just without storing a cached snippet for each page.
* The Site-framework was explicitely not used to prevent issues with SITE_ID plus the Sites-framework is not meant for this.

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -34,6 +34,7 @@ django-yubin
 mozilla-django-oidc-db
 -e git+https://bitbucket.org/maykinmedia/django-digid-eherkenning.git@0cb1978e7d628e7923dbd30ff5dbae6dbcc87129#egg=digid_eherkenning
 -e git+https://github.com/maykinmedia/python3-saml.git@58fd3ad798510c7ced110040d2fd0e6ad40d02da#egg=python3_saml
+-e git+https://github.com/maykinmedia/mozilla-django-oidc-db.git@806ba7a0448ecaae0c1271b4afa2f282601e8a6a#egg=mozilla-django-oidc-db
```
Comment (sergei-maertens) [inline requirements/base.in:37]:
```suggestion
# replace with new PyPI version once https://github.com/maykinmedia/mozilla-django-oidc-db/pull/14 is merged and released
git+https://github.com/maykinmedia/mozilla-django-oidc-db.git@806ba7a0448ecaae0c1271b4afa2f282601e8a6a#egg=mozilla-django-oidc-db
```

This also solves the problem of the package being put in the `src` directory and breaking the isort checks

File: src/openforms/multidomain/templates/multidomain/includes/switcher.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% load multidomain %}
+
+{% if sites|length > 1 %}
+    <form class="user-tools__switcher" method="GET" style="display: inline-block;">
+        <select onchange="window.location.href=this.value" title="Domain switcher">
```
Comment (sergei-maertens) [inline src/openforms/multidomain/templates/multidomain/includes/switcher.html:5]:
```suggestion
        <select onchange="window.location.href=this.value" title="{% trans 'Domain switcher' %}">
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
Minor things - I would advise to test out a dev build and then publish the OICD changes to PyPI and only then merge this PR.

**Comment by joeribekker** [comment]
> Minor things - I would advise to test out a dev build and then publish the OICD changes to PyPI and only then merge this PR.

Doing just that atm.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**